strapi/strapi
# fix(history): save status on version create

### What does it do?

- Use the actual created entry as the data stored for the version instead of the context params
- Use the content-manager's `document-metadata` service to compute the status of the document version
- 
- Update the e2e tests to account for both publish and modified statuses

Unrelated
- Make the getLocaleDictionary private

### Why is it needed?

- To correctly save and display a history version's status

### Note

- unpublish actions are broken since the middleware does not have the data that was unpublished

### How to test it?

- Go to a content-type in draft state
- Make a change and save
- Publish and save
- Make another change and save
- Go to the history page. You should see something like modified, draft, published, draft
